### PR TITLE
doc: add an example of a VM launch with a bigger root disk

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: doc/howto/instances_create.md [hunk 1]
```diff
@@ -50,16 +50,20 @@ The following examples use `lxc launch`, but you can use `lxc init` in the same
 
 ### Launch a container
 
-To launch a container with a Ubuntu 22.04 image from the `images` server using the instance name `ubuntu-container`, enter the following command:
+To launch a container with an Ubuntu 22.04 image from the `images` server using the instance name `ubuntu-container`, enter the following command:
 
     lxc launch images:ubuntu/22.04 ubuntu-container
```
Comment (tomponline) [inline doc/howto/instances_create.md:55]: Please could you update the examples to use `ubuntu:22.04`?

Comment (simondeziel) [reply doc/howto/instances_create.md:55]: Sure but if the reason was due to cloud-init resizing the rootfs automatically, that's no longer needed as even regular (!= cloud) ones from `images:` have auto partition/FS resize now.

Comment (tomponline) [reply doc/howto/instances_create.md:55]: No, my understanding was that the docs should be using the official ubuntu images.

**Comment by simondeziel** [comment]
@tomponline I replaced most but not all occurrences of `images:` by `ubuntu:`. I left a few `images:` where I felt was appropriate, let me know what you think, please.

**Review by tomponline — APPROVED** [review]
Thanks!

**Pull request merged**